src/components/Discussion/Tree/CommentList.js

```diff
@@ -284,20 +339,39 @@ const CommentNode = ({ t, comment }) => {
           <Comment.Actions
             t={t}
             comment={comment}
-            onReply={() => {
-              dispatch({ showReplyComposer: {} })
-            }}
+            onExpand={
+              board &&
+              (() => {
+                actions.fetchMoreComments({ parentId: id, after: {} })
+              })
+            }
+            onReply={
+              !board &&
```

Comment:
This is for a long standing issue that guests see the reply icon but it explodes when pressed:
```suggestion
              !board && !!displayAuthor &&
```